### PR TITLE
migrate winkernel network type string "overlay" as const

### PR DESCRIPTION
**Pull request by JornShen**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

migrate winkernel network type string "overlay" as const.
make it looks more standard.

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: pkg/proxy/winkernel/proxier.go [hunk 1]
```diff
@@ -563,7 +565,7 @@ func NewProxier(
 
 	// Network could have been detected before Remote Subnet Routes are applied or ManagementIP is updated
 	// Sleep and update the network to include new information
-	if hnsNetworkInfo.networkType == "Overlay" {
+	if hnsNetworkInfo.networkType == NETWORK_TYPE_OVERLAY {
```
Comment (feiskyer) [inline pkg/proxy/winkernel/proxier.go:568]: could you switch `==` to `strings.EqualFold()` for all places?

Comment (JornShen) [reply pkg/proxy/winkernel/proxier.go]: done.

**Comment by JornShen** [comment]
updated. PTAL 
@feiskyer

**Review by elweb9858 — APPROVED** [review]
/lgtm

**Pull request merged**